pkg/driftctl_test.go

```diff
@@ -591,9 +671,16 @@ func TestDriftctlRun_BasicFilter(t *testing.T) {
 					Attrs: &resource.Attributes{},
 				},
 			},
-			assert: func(result *test.ScanResult, err error) {
+			assert: func(t *testing.T, result *test.ScanResult, err error, store memstore.Store) {
 				result.AssertUnmanagedCount(1)
 				result.AssertResourceUnmanaged("res2", "filtered")
+
+				assert.Equal(t, "dev-dev", store.Bucket(memstore.TelemetryBucket).Get("version"))
```

Comment:
I'm not sure that we need to test telemetry data in this test case. It make sense to test store in the basic test `TestDriftctlRun_BasicBehavior` but here we want to test filter and IMHO adding memstore assertions here will lead us to more maintenance for the same scope covered. Same remark for all other test cases. BTW adding a `assertStore` attribute to testcase may be less painfull since you do not have to update every assert func in unrelated tests